.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+lib:
+  - lib/**/*
+  - include/**/*
+
+core:
```

Comment:
This label doesn't exist in GitHub yet, see also related discussion about labels at, https://trac.osgeo.org/grass/wiki/GitMigration#Migrationoftracissues

Reply:
It's `core` more like `modules`? To distinguish it from `lib(s)` or `libraries`?

Reply:
I was thinking of naming this modules, it might be a better choice.